static/styles/lib/alerts.scss

```diff
@@ -161,11 +162,10 @@
 			}
 
 			@include media-breakpoint-down(sm) {
-				left: -216px;
-
+				left: -290px;
 				&::after,
 				&::before {
-					left: 210px;
+					left: 17.8em;
```

Comment:
why overwriting a fixed px value with em value? makes it hard to find issues later